src/App.tsx

```diff
@@ -17,18 +48,36 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                todos={todos}
```

Comment:
For what purpose did you need to pass `todos` as props in `TodoFilter`? When did you use it there?